includes/class-sensei-settings-api.php

```diff
@@ -1060,6 +1061,19 @@ public function enqueue_field_styles() {
 			wp_enqueue_style( 'sensei-settings-imageselectors' );
 		}
 	}
+
+	/**
+	 * Exposes the sensei settings into the js runtime.
+	 */
+	public function expose_sensei_settings() {
+		$settings = wp_json_encode( $this->get_settings() );
+		wp_register_script( 'expose-sensei-settings', '', [], \false, \false );
+		wp_enqueue_script( 'expose-sensei-settings' );
+		wp_add_inline_script(
+			'expose-sensei-settings',
+			sprintf( 'window.senseiSettings = %s;', $settings ),
```

Comment:
We started introducing `window.sensei` in some parts. Maybe we could also use it here, like this:

`'window.sensei = window.sensei || {}; window.sensei.senseiSettings = %s;'`

Reply:
Updated. e6bbb663792477a78cb6c9afdf7ca0c9fc9217d0